src/shared/ui/date-lang/date-lang.test.tsx

```diff
@@ -11,20 +11,6 @@ describe('DateLang', () => {
     expect(screen.getByText(`${startDate}`)).toBeInTheDocument();
   });
 
-  it('renders TBD for course date in past', () => {
-    const startDate = '2020-01-01';
-
-    render(<DateLang startDate={startDate} language={[]} mode="" />);
-    expect(screen.getByText('TBD')).toBeInTheDocument();
-  });
-
-  it('renders TBD for invalid course date', () => {
-    const startDate = 'invalid-date';
-
-    render(<DateLang startDate={startDate} language={[]} mode="" />);
-    expect(screen.getByText('TBD')).toBeInTheDocument();
-  });
```

Comment:
Oh no

Reply:
It now passes TBD instead of date to component, so no need to test it inside

Reply:
done